docs/content/en/docs/usage/observability/opentelemetry.md

```diff
@@ -0,0 +1,158 @@
+
+# deploy Jaeger operator
+make jaeger-operator-k8s
+
+# after the operator is deployed, create the Jaeger instance
+make jaeger-k8s
+
+# finally, deploy the OpenTelemetry Collector
+make otel-collector-k8s
+```
+
+Find out where the Jaeger console is living. For us, we get the output:
+
+```sh
+kubectl get ingress --all-namespaces
+
+NAMESPACE       NAME           CLASS    HOSTS   ADDRESS   PORTS   AGE
+observability   jaeger-query   <none>   *                 80      54s
+```
+
+Check if the `otel-collector` service has been created:
+
+```sh
+kubectl get svc --all-namespaces
+
+NAMESPACE       NAME                        TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                                  AGE
+default         kubernetes                  ClusterIP   10.96.0.1       <none>        443/TCP                                  2m39s
+kube-system     kube-dns                    ClusterIP   10.96.0.10      <none>        53/UDP,53/TCP,9153/TCP                   2m38s
+observability   jaeger-agent                ClusterIP   None            <none>        5775/UDP,5778/TCP,6831/UDP,6832/UDP      38s
+observability   jaeger-collector            ClusterIP   10.96.238.36    <none>        9411/TCP,14250/TCP,14267/TCP,14268/TCP   38s
+observability   jaeger-collector-headless   ClusterIP   None            <none>        9411/TCP,14250/TCP,14267/TCP,14268/TCP   38s
+observability   jaeger-operator-metrics     ClusterIP   10.96.18.77     <none>        8383/TCP,8686/TCP                        38s
+observability   jaeger-query                ClusterIP   10.96.94.227    <none>        16686/TCP,16685/TCP                      38s
+observability   otel-collector              NodePort    10.96.141.105   <none>        4317:30080/TCP,8889:31521/TCP            50s
+```
+
+Now, setup a port forward to the `otel-collector` service:
+
+```sh
+kubectl port-forward service/jaeger-query -n observability 8080:16686 &
+```
+
+You should now be able to access Jaeger at [http://localhost:8080/](http://localhost:8080/).
+
+### Installing Fission
+
+At the time of writing this document, the Fission installation does not have OpenTelemetry enabled by default.
+In order to enable OpenTelemetry, we need to explicitly set the value of `otelCollectorEndpoint`:
+
+```sh
+export FISSION_NAMESPACE=fission
+helm install --version 1.14.1 --namespace $FISSION_NAMESPACE \
+  fission fission-charts/fission-all \
+  --set openTracing.enabled=false \
+  --set otelCollectorEndpoint="otel-collector.observability.svc:4317"
+```
+
+Note: You may have to change the `otelCollectorEndpoint` value as per your setup.
+
+Currently, Fission supports both - OpenTracing (added a while ago) and OpenTelemetry (new addition).
+In a future release, support for OpenTracing will be removed and OpenTelemetry will be used by default.
+
+## Testing
+
+In order to verify that our setup is working and we are able to receive traces, we will deploy and test a fission function.
+
+### NextJS App
```

Comment:
We can use the sample nodejs(hello) function here so that users do not need to lot of setup to get example working.